types/src/proof/mod.rs

```diff
@@ -453,10 +457,15 @@ pub fn verify_sparse_merkle_element(
                  the only existing key, if it existed. So this is not a valid non-inclusion proof."
             );
         }
-        (None, None) => (),
+        (None, None) => {
+            // This is a non-inclusion proof.
+            // The proof intends to show that if a leaf node representing `element_key` is inserted,
+            // it will show up at a currently empty position.
+            // `sibling` should prove the route from this empty position to the root.
+        }
     }
 
-    let leaf_hash = match sparse_merkle_proof.leaf() {
+    let current_hash = match sparse_merkle_proof.leaf() {
```

Comment:
@msmouse why change this one? It is the leaf hash.

Reply:
I think @msmouse would argue some "leaf" is not real leaf but just placeholder hash representing an empty subtree.